plugins/blockvault_client_plugin/blockvault_client_plugin.cpp

```diff
@@ -44,6 +43,9 @@ void blockvault_client_plugin::plugin_initialize(const variables_map& options) {
             elog("unknown block-vault-backend option, skipping it");
          }
       }
+      else {
+         ilog("no block-vault-backend option is specified, skipping it");
+      }
```

Comment:
Ideally the plugin would not initialize at all if not specified. But chain_plugin requires blockvault_client_plugin currently. Would be great if that was not the case.

Reply:
If we are going to make that change, we need to do that in a Jira task just for that specifically.